docs/basics/basic_knowledge.md

```diff
@@ -0,0 +1,16 @@
+# Basic Knowledge
+
+## Benchmark Structure
+
+In order to successfully create a benchmark, keep in mind the following basic benchmark structure in YAML file.
+
+```
+Global Configuration
+Phases
+  Scenarios
+    Steps
+    HTTP Request (Note: httpRequest is a mandatory Step)
```

Comment:
httpRequest is *not* a mandatory step. You can have a perfectly valid scenario not doing any requests at all, just not really useful. You can have multiple steps as well.

Also, the way you formulated this looks as if `httpRequest` was something different than the other steps. What about

```
Global Configuration - HTTP configuration, agents...
Phases
  Scenarios
    Sequences (yes, you've missed them) - independent control flows in the scenario
      Steps - setting variables, firing requests, waiting for conditions...
        Step properties - parameters, handlers...
```
(feel free to rephrase)